### PR TITLE
Reader: Update Post Notifications popover UI

### LINKED ISSUE #19091: Reader: Settings popover moves after toggling new posts/comments switch
Video: https://cloudup.com/cszPvEb-mzu

This happens in both Manage and fullpost.

### PR DESCRIPTION
**Pull request by jancavan**
This PR:

- Adds bell icon to Post Notifications.
- Adds dividers between popover options.
- Fixes Settings button minor misalignment.

Fixes #19091.

**Before:**
screenshot 2018-01-15 20 54 58

**After:**
screenshot 2018-01-15 20 56 36

### CODE REVIEW

**Comment by jancavan** [comment]
@bluefuton Is there any reason why we're hiding the microcopy under Post Notifications? I think we're already able to receive both web and mobile notifications?

**Comment by bluefuton** [comment]
> Is there any reason why we're hiding the microcopy under Post Notifications?

For one reason only - the appearance was a bit wonky and I didn't have time to style it :) Shall we add it in on this PR?

**Comment by blowery** [comment]
Nice work @jancavan!

File: client/blocks/reader-site-notification-settings/index.jsx [hunk 1]
```diff
@@ -122,19 +122,20 @@ class ReaderSiteNotificationSettings extends Component {
...
+						{ translate( 'Notify me of new posts' ) }
+						<Gridicon icon="bell" size="18" />
 						<FormToggle
 							onChange={ this.toggleNewPostNotification }
 							checked={ sendNewPostsByNotification }
 							wrapperClassName="reader-site-notification-settings__popout-form-toggle"
 						/>
 						<p className="reader-site-notification-settings__popout-hint">
-							{ translate( 'Receive web and mobile notifications for new posts from this site' ) }
+							{ translate( 'Receive web and mobile notifications for new posts from this site.' ) }
```
Comment (a8ci18n) [inline client/blocks/reader-site-notification-settings/index.jsx]: :ok: This change is safe, we'll keep the existing 16 translations.

File: client/blocks/reader-site-notification-settings/index.jsx [hunk 2]
```diff
@@ -122,19 +122,20 @@ class ReaderSiteNotificationSettings extends Component {
 					className="reader-site-notification-settings__popout"
 				>
 					<div className="reader-site-notification-settings__popout-toggle">
-						{ translate( 'New post notifications' ) }
+						{ translate( 'Notify me of new posts' ) }
```
Comment (a8ci18n) [inline client/blocks/reader-site-notification-settings/index.jsx]: Hi! I've found a possible matching string that has already been translated **66** times:
`translate( 'Notify me of new posts via email.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by bluefuton** [comment]
Popover looks great @jancavan! I gave it a spin in Chrome and Safari 👍 

I wonder if we can find a method of aligning the Settings link that works across languages? It lines up nicely in English now but in German it's a bit wonky:

screen shot 2018-01-17 at 13 04 22

**Comment by bluefuton** [comment]
This also appears to fix #19091.

**Comment by blowery** [comment]
> I wonder if we can find a method of aligning the Settings link that works across languages? It lines up nicely in English now but in German it's a bit wonky:

Do we want to hold this PR for that, or address it in another PR?

**Comment by blowery** [comment]
Seeing an error in the console

```
index.jsx:97 Warning: Failed prop type: Invalid prop `size` of type `string` supplied to `t`, expected `number`.
    in t (created by ReaderSiteNotificationSettings)
    in ReaderSiteNotificationSettings (created by Localized(ReaderSiteNotificationSettings))
    in Localized(ReaderSiteNotificationSettings) (created by Connect(Localized(ReaderSiteNotificationSettings)))
    in Connect(Localized(ReaderSiteNotificationSettings)) (created by FeedHeader)
    in div (created by FeedHeader)
    in div (created by FeedHeader)
    in div (created by FeedHeader)
    in div (created by FeedHeader)
    in FeedHeader (created by Localized(FeedHeader))
```

when hitting http://calypso.localhost:3000/read/feeds/20463208

**Comment by jancavan** [comment]
Good catch @bluefuton 👀

**Comment by blowery** [comment]
fixed the console warning in https://github.com/Automattic/wp-calypso/pull/21551/commits/afa28aa1caeab7fe57da11e49e5ae57a9601076a

**Comment by jancavan** [comment]
> Do we want to hold this PR for that, or address it in another PR?

Actually, I'll do it in a different PR.

**Pull request merged**